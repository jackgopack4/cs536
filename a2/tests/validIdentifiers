_
__
_123
_ab
_a123b
Hello
boolean
integer
returning
elsewhile
