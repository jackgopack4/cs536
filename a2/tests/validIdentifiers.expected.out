Token: ID
LineNum: 1
CharNum: 1
Value: _

Token: ID
LineNum: 2
CharNum: 1
Value: __

Token: ID
LineNum: 3
CharNum: 1
Value: _123

Token: ID
LineNum: 4
CharNum: 1
Value: _ab

Token: ID
LineNum: 5
CharNum: 1
Value: _a123b

Token: ID
LineNum: 6
CharNum: 1
Value: Hello

Token: ID
LineNum: 7
CharNum: 1
Value: boolean

Token: ID
LineNum: 8
CharNum: 1
Value: integer

Token: ID
LineNum: 9
CharNum: 1
Value: returning

Token: ID
LineNum: 10
CharNum: 1
Value: elsewhile

